### ISSUE TITLE
Persistance

### ISSUE THREAD

**Issue by shfsicig** [beigeworm/PoshCord-C2#3]
AddPersistance should be enabled by default keep the commands available but just call AddPersistance function on run

**Comment by beigeworm** [comment]
Yes I will add a switch to app persistance on run - however i wont enable it by default as windows is much more likely to detect the saved script to disk along with the vbscript stager.. thus making the whole script more open to detection  - if you want to enable by default, fork and host this repo once i've made the chages
